bin/cheritest/cheritest.c

```diff
@@ -136,11 +141,13 @@ static const struct cheri_test cheri_tests[] = {
 	  .ct_desc = "Print out a list of CP2 registers and values",
 	  .ct_func = test_listregs,
 	  .ct_flags = CT_FLAG_STDOUT_IGNORE },
+#endif
 
 	/*
 	 * Capability manipulation and use tests that sometimes generate
 	 * signals.
 	 */
+#ifdef __mips__
```

Comment:
I guess for now this is fine, but most of these should also work for RISC-V, we just have to update the exccode value.

Reply:
Yeah, I just started with a blunt hammer.  We need to de-mips most of the trapping tests.